src/frontend/context/AppProvider.js

```diff
@@ -1,29 +1,28 @@
-// @flow
 import * as React from "react";
 
 import { LocationProvider } from "./LocationContext";
 import { ObservationsProvider } from "./ObservationsContext";
 import { ConfigProvider } from "./ConfigContext";
 import { SettingsProvider } from "./SettingsContext";
 import { DraftObservationProvider } from "./DraftObservationContext";
-import { SecurityProvider } from "../screens/Security/SecurityContext";
+import { SecurityProvider } from "./SecurityContext";
 import { MapStyleProvider } from "./MapStyleContext";
 
 // This is a convenience wrapper for providing all App contexts to the tree,
 // apart from the Permissions Provider which is needed separately.
 const AppProvider = ({ children }: { children: React.Node }) => (
   <LocationProvider>
-    <ObservationsProvider>
-      <ConfigProvider>
-        <SettingsProvider>
-          <SecurityProvider>
+    <SecurityProvider>
```

Comment:
I moved the `SecurityProvider` outside the `ObservationProvider` because in obscure mode, no observations will be seen, so the `observationProvider` needs to know if obscure mode is on